plugin/orderfield.go

```diff
@@ -0,0 +1,40 @@
+package plugin
+
+import (
+	"github.com/gogo/protobuf/protoc-gen-gogo/generator"
+)
+
+type OrmOrderField struct{}
+
+func (o *OrmOrderField) getOrderedFieldNames(ormable *OrmableType, message *generator.Descriptor) (fields []string) {
+
+	for _, v := range message.GetField() {
+		fieldName := generator.CamelCase(*v.Name)
+		fields = append(fields, fieldName)
+	}
+
+	ormFields := o.getOrmableFieldNames(ormable.Fields)
```

Comment:
Why not just iterate over `ormable.Fields`? Why need `getOrmableFieldNames` function?
```golang
for ormField := range ormable.Fields {
    ...
}
```

Reply:
@dkukharau thanks. Will do iterate over `ormable.Fields`